packages/quantic/cypress/integration/document-suggestion/document-suggestion.cypress.ts

```diff
@@ -420,4 +421,41 @@ describe('quantic-document-suggestion', () => {
       });
     });
   });
+
+  describe('when two document suggestions have a similar title', () => {
+    it('should render the component and all parts as expected', () => {
+      visitDocumentSuggestion();
+
+      scope('when loading the page', () => {
+        Expect.displayAccordion(false);
+        Expect.numberOfSuggestions(0);
+        Expect.displayNoSuggestions(true);
+      });
+
+      scope('when fetching suggestions', () => {
+        mockDocumentSuggestion(similarDocuments);
+        fetchSuggestions();
+        Expect.displayNoSuggestions(false);
+        Expect.displayAccordion(true);
+        Expect.numberOfSuggestions(similarDocuments.length);
+        Expect.displayAccordionSectionContent(true, 0);
+      });
+
+      scope('when clicking on a document suggestion', () => {
+        const clickIndex = 1;
+
+        Actions.clickSuggestion(clickIndex);
+        Expect.logClickingSuggestion(clickIndex, similarDocuments);
+        Expect.displayAccordionSectionContent(true, 0);
+        Expect.displayAccordionSectionContent(true, clickIndex);
```

Comment:
Am I understanding correctly that, when many suggestions have the same title, clicking a suggestion automatically unfold all suggestions sharing the same title?

Reply:
I just updated the PR description with two video recordings to make it clear what's the bug here and what's the expected behaviour. Also made a small change on the e2e test to make it more clear.